### PR TITLE
linux: add support for MIPS (fix uv_cpu_info())

### PR DESCRIPTION
**Pull request by bsnote**

### CODE REVIEW

File: src/unix/linux-syscalls.c [hunk 1]
```diff
@@ -37,7 +37,17 @@
 # else
 #  define UV_SYSCALL_BASE 0x900000
 # endif
-#endif /* __arm__ */
+#endif	/* __arm__ */
+
+#if defined(__mips__)
+# if _MIPS_SIM == _MIPS_SIM_ABI32
+#  define __NR_Linux 4000
```
Comment (bnoordhuis) [inline src/unix/linux-syscalls.c]: This conflicts with the #define from unistd.h, doesn't it?

File: src/unix/linux-core.c [hunk 2]
```diff
@@ -438,7 +438,7 @@ static int read_models(unsigned int numcpus, uv_cpu_info_t* ci) {
   model_idx = 0;
   speed_idx = 0;
 
-#if defined(__arm__) || defined(__i386__) || defined(__x86_64__)
+#if defined(__arm__) || defined(__i386__) || defined(__mips__) || defined(__x86_64__)
```
Comment (bnoordhuis) [inline src/unix/linux-core.c]: Long line, please wrap at 80 columns like this:

``` c
#if defined(__arm__) || \
    defined(__i386__) || \
// etc.
```

**Comment by bsnote** [comment]
Thanks Ben. Fixed now.

**Comment by bnoordhuis** [comment]
Landed in 3d251ad, thanks!

**Comment by bnoordhuis** [comment]
Wait, make that 5096f1e. Landing it in v0.10 rather than master.

**Pull request closed without merging**